DEVELOPMENT.md

```diff
@@ -126,3 +126,39 @@ On Fedora, this can be fixed by adding a `.conf` file to `/etc/sysctl.d/ increas
 fs.inotify.max_user_watches=2097152
 fs.inotify.max_user_instances=256
 ```
+
+### Error: can only create exec sessions on running containers: container state improper
+
+This may happen when the containers have stopped. This can be fixed by deleting the already created clusters and starting the script again.
+
+```bash
+sudo kind delete cluster us-east1 us-west1
```

Comment:
Shouldn't it be `kind delete clusters us-west1 us-east1 `?

Reply:
Ah, yes, changing. Thanks 😅